### ISSUE TITLE
conflicting repo name

### ISSUE THREAD

**Issue by chaserx** [openlexington/openlexington.github.io#1]
Do not put a CNAME file in here. It will fuck with the existing CNAME DNS magic applied to the openlexington.org repo. 

If this is for finding ESL classes, cool. Let's rename, please.

**Comment by JunHuang01** [comment]
Gottcha, will do, this was put here to see if I can have it up and running.

Everything is all ready, so just feel free to move it anywhere.

**Comment by JunHuang01** [comment]
it can be put under openlexington.org like esl.openlexington.org or www.openlexington.org/esl

**Comment by chaserx** [comment]
yeah. let's do that. use the existing ESL repo, toggle GH pages there, add CNAME file to something like esl.openlexington.org and I'll add appropriate DNS entries. See Lex_to_LIVES repo for pattern.

**Comment by chaserx** [comment]
functional: https://openlexington.github.io/esl/

we'll just add CNAME business.
